tests/test_downloadermiddleware_cookies.py

```diff
@@ -646,7 +646,7 @@ def test_user_set_cookie_domain_suffix_public_period(self):
             "https://foo.co.uk",
             "https://bar.co.uk",
             "co.uk",
-            cookies1=False,
+            cookies1=True,
```

Comment:
With the modification made on the handler, I think this test should change since the cookie will be present on the first request.